src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -4107,7 +4107,8 @@ private object ExecuteQuery<TProjection>()
             var finalQuery = ((DocumentQuery<T>)DocumentQuery).CreateDocumentQueryInternal<TProjection>(
                 new QueryData(fields, projections, FromAlias, _declareTokens, _loadTokens, _declareTokens != null || _jsSelectBody != null)
                 {
-                    IsProjectInto = _isProjectInto
+                    IsProjectInto = _isProjectInto,
+                    QueryStatistics =  _queryStatistics
```

Comment:
I haven't found any cases for this, but I noticed that other methods creating `QueryData` also pass the `QueryStatistics`